app/web/components/AppRoute.tsx

```diff
@@ -47,6 +48,7 @@ export const useAppRouteStyles = makeStyles((theme) => ({
   "@global html": {
     scrollPaddingTop: `calc(${theme.shape.navPaddingXs} + ${theme.spacing(2)})`,
     height: "100%",
+    overflow: "hidden",
```

Comment:
FYI both of these are fixing an issue I noticed where the whole app scrolls. We don't want that for mobile nor regular app (i.e. the nav bar scrolling with the whole page). Top layer should not scroll, that should be responsibility for the child components.

Example of what I mean on mobile (also relevant to non-Mobile):

![Uploading ScreenRecording2024-10-27at11.38.40AM-ezgif.com-video-to-gif-converter.gif…]()